### ISSUE TITLE
Cannot use hyphen in symfony asset package name

### ISSUE THREAD

**Issue by damaya** [symfony/symfony#28122]
**Symfony version(s) affected**: 3.4.14

**Description**  
When using hyphen in symfony asset package name, the following error is shown: 
>"An exception has been thrown during the rendering of a template ("There is no "frontend-app" asset package.")."

**How to reproduce**  
``` yaml
framework:
        packages:
            frontend-app:
                base_path: '/'
```
NOTE: in this example, when removing the hyphen or replacing hyphen with underscore, it works as expected, for example:
``` yaml
framework:
        packages:
            frontendapp:
                base_path: '/'
```
or this
``` yaml
framework:
        packages:
            frontend_app:
                base_path: '/'
```

**Comment by dmaicher** [comment]
I guess the keys are normalized when processing the config:

> normalizeKeys(false)
    If called (with false), keys with dashes are not normalized to underscores. It is recommended to use this with prototype nodes where the user will define a key-value map, to avoid an unnecessary transformation.

http://symfony.com/doc/current/components/config/definition.html#array-node-options

@damaya can you check if adding `->normalizeKeys(false)` helps here?

https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L634

**Comment by damaya** [comment]
Hi @dmaicher thanks for your feedback about this, yes, this is clear now, i tested wit the 
`->arrayNode('packages')->normalizeKeys(false)` and it worked, i suppose this would't require any update for the array mappings yml configs. 
Thanks so much.

**Comment by dmaicher** [comment]
Up for creating a PR with this fix? :) I cannot think of any reason not to do it that way.

**Comment by javiereguiluz** [comment]
Closing as fixed by #30007.
